tests/unit/ConsentContinuationOut.spec.ts

```diff
@@ -194,14 +190,8 @@ describe('Consent to Continuation Out view', () => {
     // wait for fetch to complete
     await flushPromises()
 
-    const button = wrapper.find('#consent-save-btn')
-    expect(button.attributes('disabled')).toBeUndefined()
-
-    // click the Save button
-    await button.trigger('click')
-
-    // wait for save to complete and everything to update
-    await flushPromises()
+    // call the save action (since clicking button doesn't work)
+    await vm.onClickSave()
 
     // verify new Filing ID
     expect(vm.filingId).toBe(456)
```

Comment:
This looked like it worked because it was using the filing that was (incorrectly) on the route. When I fixed this, the filing ID became 0 (not 456), which is how I discovered that the save action was never called.